src/results/parse.jl

```diff
@@ -258,6 +263,7 @@ function parse_power_results!(config, data)
     add_table_col!(data, :bus, :pflow_in, pflow_in_bus, MWFlow,"Average power flowing into this bus, positive")
     add_table_col!(data, :bus, :plserv, plserv_bus, MWServed,"Average power served at this bus")
     add_table_col!(data, :bus, :plcurt, plcurt_bus, MWCurtailed,"Average power curtailed at this bus")
+    add_table_col!(data, :bus, :val_plcurt, val_plcurt_bus, Dollars, "Value of power curtailed at this bus")
```

Comment:
after thinking about it more, I think using "cost" instead of "value" will be more clear, can you update this to be "plcurt_cost" or something like that?